config/development.json

```diff
@@ -118,6 +118,7 @@
 		"post-editor/image-editor": true,
 		"post-editor/media-advanced": false,
 		"press-this": true,
+		"publicize/preview": true,
```

Comment:
This is good to ship after we flip this to false. We can turn it on for dev once the button is wired up.